Newsletters: add 324 (2024-10-11)

harding opened the pull request:
- [x] Add btcd security news item @harding 
- [x] Add summary and link to #1937 blog post if ready by Thursday evening
- [x] Add lede, update releases/RCs, add topic entries @harding 
- [x] Bitcoin Core PR Review Club @stickies-v 
- [x] Merge summaries @Gustavojfe

jonatack commented on _posts/en/newsletters/2024-10-11-newsletter.md:
```diff
@@ -0,0 +1,170 @@
+the meeting.*
+
+[add getorphantxs][review club 30793] is a PR by [tdb3][gh tdb3] that
+adds a new experimental RPC method named `getorphantxs`. Since it is
```
```suggestion
adds an experimental `getorphantxs` RPC. Since it is
```

harding replied:
Leaving as-is.  I think both phrasings are fine and this isn't a section I wrote, so I'd prefer to let the original author's version stand.

jonatack commented on _posts/en/newsletters/2024-10-11-newsletter.md:
```diff
@@ -0,0 +1,170 @@
+[add getorphantxs][review club 30793] is a PR by [tdb3][gh tdb3] that
+adds a new experimental RPC method named `getorphantxs`. Since it is
+intended primarily for developers, it is hidden. This new method
+provides the caller with a list of all current orphan transactions,
```
```suggestion
This new RPC returns a list of all current orphan transactions (optionally with different levels of verbosity),
```

harding replied:
Leaving as-is.  I think both phrasings are fine and this isn't a section I wrote, so I'd prefer to let the original author's version stand.

harding commented:
Force pushed so I could add a link to the Wallet Guide, which was on master.

LarryRuane commented on _posts/en/newsletters/2024-10-11-newsletter.md:
```suggestion
    However, during a surge in network activity in May 2023, several
```
(don't want "during" twice)

LarryRuane commented on _posts/en/newsletters/2024-10-11-newsletter.md:
```suggestion
    Developer 0xB10C determined that the CPU was being
```

LarryRuane commented on _posts/en/newsletters/2024-10-11-newsletter.md:
```suggestion
    Pseudonymous developer 0xB10C was able to determine that the CPU was being
```
(but maybe not, because consistency may then require updates in many other places)

harding replied:
Leaving as-is.  We've been inconsistent about this in the past, for which I apologize, but going forward I don't think it's necessary.  We insert the title "developer" out of necessity to prevent confusion between 0xB10C's nym and conflicting phrasing (e.g. a hex string).  Inserting anything else seems like a distraction.  It's also not possible for us know for certain that someone is (or isn't) using a nym; for all I know "Larry Ruane" is your nym and "0xB10C" is that person's legal name.

LarryRuane commented on _posts/en/newsletters/2024-10-11-newsletter.md:
```suggestion
    ensuring transactions left the queue more quickly, especially
```

LarryRuane commented on _posts/en/newsletters/2024-10-11-newsletter.md:
```suggestion
  commit to itself, so signers commit to all the data in the signature
```

LarryRuane commented on _posts/en/newsletters/2024-10-11-newsletter.md:
```suggestion
  result in a significant loss of money.
```
or "significant losses of money" but I prefer the above.

LarryRuane commented on _posts/en/newsletters/2024-10-11-newsletter.md:
```suggestion
  create or broadcast transactions.  The guide walks through the use
```

LarryRuane commented on _posts/en/newsletters/2024-10-11-newsletter.md:
```suggestion
  limit to how many orphans are retained? Is there a limit to how
  long orphans can stay in the orphanage?"
```

harding commented:
Made edits (or left comment replies) to all feedback; thanks @jonatack and @LarryRuane.  Added late-breaking security news items and blub for @instagibbs  Wallet Guide; I'm sorry they're wasn't much time for review (but extra thanks to @LarryRuane for providing the last-minute review); @bitschmidty please feel free to make any edits you deem necessary before merge.  Added lede, updated releases/RCs, and added topic entries.  Reviewed and made some small edits to @stickies-v and @Gustavojfe sections (thanks!).

Thanks everyone!

darosior commented on _posts/en/newsletters/2024-10-11-newsletter.md:
```suggestion
  script than Bitcoin Core would remove before it respectively
```

?

bitschmidty commented on _topics/en/testnet.md:
```diff
@@ -106,6 +106,9 @@ optech_mentions:
   - title: "New CPUNet testnet announced with a modified PoW algorithm designed for CPU-only mining"
     url: /en/newsletters/2024/08/23/#cpunet-testnet-announced
 
+  - title: "Rust Bitcoin #2945 introduces support for Testnet4"
```
```suggestion
  - title: "Rust Bitcoin #2945 introduces support for testnet4"
```

bitschmidty commented on _posts/en/newsletters/2024-10-11-newsletter.md:
```suggestion
- [Rust Bitcoin #2945][] introduces support for [testnet4][topic testnet] by
```

bitschmidty commented on _posts/en/newsletters/2024-10-11-newsletter.md:
```suggestion
  necessary parameters and blockchain constants for testnet4.
```

bitschmidty commented on _topics/en/ephemeral-anchors.md:
```diff
@@ -72,6 +72,10 @@ optech_mentions:
   - title: "Bitcoin Core PR Review Club for #30352: Add PayToAnchor (P2A) as standard output script for spending"
     url: /en/newsletters/2024/08/09/#bitcoin-core-pr-review-club
 
+  - title: "Guide for Wallets Employing Bitcoin Core 28.0 Policies: P2A output scripts"
```
Thanks for adding this topic links for the guide blog post, @harding !

bitschmidty reviewed (commented):
Just a couple small items

bitschmidty commented:
Took suggestions from @darosior and @vostrnad, squashed.

Thank you to @harding @Gustavojfe and @stickies-v for authoring and @LarryRuane @jonatack @vostrnad @darosior for reviewing! 🚀

jonatack commented on _posts/en/newsletters/2024-10-11-newsletter.md:
Ought the Musig optech topic be updated with this entry?

bitschmidty replied:
I think so. What to take a crack at it?

jonatack replied:
Proposed in https://github.com/bitcoinops/bitcoinops.github.io/pull/1957.
